lib/rules.json

```diff
@@ -856,6 +856,7 @@
                             "divClassHead": true,
                             "logo": true,
                             "title": true,
+                            "h3CrType": true,
```

Comment:
This should be moved *after* "dateTitleH2" so it keeps the same order as on the document.